Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,11 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  return arr.reduce(
+    (uniquesArr, currentElm) =>
+      uniquesArr.includes(currentElm) ? uniquesArr : [...uniquesArr, currentElm],
+    [],
+  );
```

Comment:
Would suggest that you had a look at a new topic in JavaScript called `Sets` 🙂

Reply:
Great, learned a brand new subject.